docs/interactions/Message_Components.md

```diff
@@ -130,6 +134,48 @@ Buttons come in a variety of styles to convey different types of actions. These
 
 When a user clicks on a button, your app will receive an [interaction](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object) including the message the button was on:
 
+## Input Text
+
+Input text is a modal component accepting text input from users.
+
+- `input_text` must be sent inside an Action Row
+- An Action Row can have exactly 1 `input_text` component
+
+### Input Text Object
+
+###### Input Text Structure
+
+| Field        | Type                                                | Description                                                                                           |
+| ------------ | ------- | ----------------------------------------------------------------------------------------------------- |
+| type         | integer | `4` for a text input field                                                                            |
+| style        | integer | one of [input text styles](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/input-text-object-input-text-styles) |
+| label        | string  | text that appears on top of the input text field, max 80 characters                                   |
+| placeholder? | string  | placeholder for the text input                                                                        |
+| min_length?  | integer | minimal length of text input                                                                          |
+| max_length?  | integer | maximal length of text input                                                                          |
```

Comment:
Why custom_id isn't here, but it's in the json sample?

Reply:
Thank you, you're right.
Fixed that + the description of `custom_id` on the select component fields.